oio/account/backend.py

```diff
@@ -581,7 +581,7 @@ def cast_fields(self, info):
                 pass
         for what in (BUCKET_PROP_REPLI_ENABLED.encode('utf-8'), ):
             try:
-                decoded = info.get(what, b'').decode('utf-8')
+                decoded = (info.get(what) or b'').decode('utf-8')
                 info[what] = boolean_value(decoded)
```

Comment:
I'm not sure this fixes anything.
```
>>> from oio.common.easy_value import boolean_value
>>> boolean_value(b''.decode('utf-8'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/fvenneti/src/public_git/oio-sds/oio/common/easy_value.py", line 60, in boolean_value
    raise ValueError('Boolean value expected')
ValueError: Boolean value expected
```

```suggestion
                decoded = (info.get(what) or b'').decode('utf-8') or None
                info[what] = boolean_value(decoded)
```
Why `None`? Because it is the first value considered "false" by the `boolean_value` method.